schemas/data/poi-entries.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/placecontext/poi-entries",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "poi-entries",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/metric"],
+  "description":
+    "The place context metric poi-entries describes the number of times a person has left a Point of Interest (POI).",
+  "definitions": {
+    "metric": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "const": "xdm:poiEntries",
+          "description": "The unique identifier of this metric."
+        },
+        "schema:name": {
+          "type": "string",
+          "const": "placecontext: poiEntries",
+          "description":
+            "The human-readable name of the metric. The name can be used in user interfaces and does not have to be unique."
+        },
+        "xdm:measurement": {
+          "type": "string",
+          "description": "How to take measures of this metric.",
+          "const": "count"
+        },
+        "xdm:unit": {
```

Comment:
What is the point of this if the only value is NULL?

Reply:
It is just the pattern that Lars decided on for measures, they need to have a unit, and in some cases there is no unit, it is just a count, in other places it is monetary or a weight.

Reply:
You can omit `xdm:unit` if the value is `null`. Both variants are valid.